open-metadata-implementation/user-interfaces/presentation-server/README.md

```diff
@@ -11,50 +11,74 @@ servers.
 
 ## For development
  * Add the presentation-server folder into [Visual Studio Code](https://code.visualstudio.com/). It will create a new workspace.
- * You can open a terminal in Visual Studio code to issue the npm commands
+ * You can open a terminal in Visual Studio Code to issue the npm commands
  * Set up your environment by running `npm install`
  * Developers can specify the environment variable(s) in a .env file, by copying [.env_sample](nodejs/.env_sample) to a file
  called .env in the same folder and amend the values as required.
  * Once the .env file is in place, build and start the server using `npm start`. Use other npm parameters as specified in [package.json](nodejs/package.json).  
 
-## FOR NEW DEVELOPMENT (as of 8/2/2020)
- * Currently, the UI is not optimized to run in development mode. Developers must rebuild the static files every time a change is made.
- * Because of this, work is currently underway to remake the client with Create React App. This will be done iteratively, and updates posted here.
- * Old client code will be maintained until the cutover is complete, so developers can run the traditional way if desired.
- * The new client is maintained in the cra-client directory.
- * Set up your environment by running `yarn`.
- * Enter `http://localhost:3000/` to access the new UI in development mode.
- 
 ## Access UI using the browser. 
- * Enter 'https://localhost:8091/<tenant-name>/' on the UI to access the tenanted UI (<tenant-name> is the serverName used by the presentation server). The Ui will prompt for a login; some of the 
+ * Enter 'https://localhost:8091/<tenant-name>/login' on the UI to access the tenanted UI (<tenant-name> is the serverName used by the presentation server). The Ui will prompt for a login; some of the 
  Coco Pharmaceutical personnas have been enabled here - use user 'faith' and password 'admin'. If there is an environment variable :
   ` EGERIA_PRESENTATIONSERVER_SERVER_aaa={"remoteServerName":"cocoView1","remoteURL":"https://localhost:9443"}`
   then the browser url to use to login is `https://localhost:8091/aaa`. All browser requests for this tenant will be issued using urls starting
   `https://localhost:8091/aaa`. 
-    
-  
+
+## There is on-going work to enhance the development experience 
+ * The build described above is a production build; meaning everything is rebuilt each time a change is made.
+ * There is ongoing work to allow developers to do be able to make changes and hot swap them into a running server.
+ * The ongoing work is being tracked in issue [Git issue 3543](https://github.com/odpi/egeria/issues/3543)
+ * The work involves using Create React App. This will be done iteratively.
+ * The current latest client code is in the client folder.
+ * The new work is being developed in the cra-client directory and may contain out of date Presentation Server source code. 
+
+## There is ongoing work to enhance the ability to easily run the new services in a demo environment
```

Comment:
There is ongoing work to enable easy running of the new view services in a demo environment

Reply:
I changed to 'Easy running of the new Presentation Server and view services in a demo environment' as the text was rather long when formatted. The text in this para already refers to work in progress, plus it's really PS that gives the end user function